www/directives/player/player.html

```diff
@@ -1,10 +1,10 @@
 <div class='container-fluid'>
-	<panel-404 ng-if='playerCtrl.error' />
+	<panel-404 ng-if='vm.error' />
```

Comment:
I'm not against the `vm` controller name but in my experience, having all your controllers named `vm` makes a huge deal with nested controllers.